charts/behave/Chart.yaml

```diff
@@ -26,10 +26,16 @@ dependencies:
     repository: https://strg-at.github.io/charts/
     condition: behave-content-indexer.enabled
     version: 1.0.0
+  - name: behave-crawler
```

Comment:
Chart Version in this file (on top) need to be increased whenever this file change